### PR TITLE
refactor(flash): move remoteAddr to options bag

### PR DESCRIPTION
**Pull request by crowlKats**
Applies suggestion from #17912

### CODE REVIEW

File: ext/flash/01_http.js [hunk 1]
```diff
@@ -569,16 +569,19 @@ function createServe(opFn) {
...
-                const { 0: hostname, 1: port } = core.ops.op_flash_addr(
-                  serverId,
-                  i,
-                );
-                return {
-                  hostname,
-                  port,
-                };
+              resp = handler(req, {
+                remoteAddr() {
```
Comment (aapoalas) [inline ext/flash/01_http.js:575]: Missing `get`? Type suggests plain value but code is a function call.

Comment (crowlKats) [reply ext/flash/01_http.js:575]:
```suggestion
                get remoteAddr() {
```

**Review by lucacasonato — APPROVED** [review]
LGTM. @bartlomieju ptal also

**Review by bartlomieju — APPROVED** [review]
LGTM too

**Pull request merged**